### ISSUE TITLE
Expo Client App closes on addition of accessibility value in IOS Simulator

### ISSUE THREAD

**Issue by themmyloluwaa** [expo/expo#10019]
## 🐛 Bug Report
Yes
### Summary of Issue 
The addition of accessibilityValue props on a component causes the expo client app to close in the IOS simulator when you test with the accessibility inspector.

### Environment - output of `expo diagnostics` & the platform(s) you're targeting
The platform is IOS and I am running on an Iphone 11 Pro Max (IOS 13.5 OS)

### Reproducible Demo
Snack
https://snack.expo.io/@codekagei/accessibilityvalue-bug

Repo.

https://github.com/themmyloluwaa/activity-tracker

 
### Steps to Reproduce
Fork the repo provided above.

 Checkout to accessibility-value branch.

Proceed to HeadingComponent and uncomment the accessibility value comment.

Run the project with expo start.

Test with the accessibility inspector.

### Expected Behavior vs Actual Behavior

The value of the component should be available.
Actual behavior is the expo client keeps closing upon getting to that component.

**Comment by byCedric** [comment]
Hi @themmyloluwaa! The `accessibilityValue` should be an object, not just a string. If you use `accessibilityValue={{ text: 'whatever' }}`, that should work. For more info, [see the docs](https://reactnative.dev/docs/accessibility#accessibilityvalue). 

Hope this helps!

**Comment by themmyloluwaa** [comment]
Yes, it does. Thank you.
